chia/full_node/full_node.py

```diff
@@ -212,7 +213,7 @@ def sql_trace_callback(req: str):
                 async with conn.execute(
                     "SELECT name FROM sqlite_master WHERE type='table' AND name='full_blocks'"
                 ) as cur:
-                    if len(await cur.fetchall()) == 0:
+                    if len(list(await cur.fetchall())) == 0:
```

Comment:
Though perhaps the query could cover this better?

```suggestion
                    if await cur.fetchone() is None:
```

Reply:
This is not the focus of the PR, I want to avoid making any real code changes which add risk.